packages/Stylers/stories/Stylers.sass.stories.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+
+import React from "react";
+import { Story, Rule } from "storybook/assets/styles/common.styles";
+import { getStoryName } from "storybook/storyTree";
+import storyParameters from "storybook/assets/storyParameters";
+import { Box } from "./Stylers.stories.styles";
+import "./helpers.scss";
+
+const storyName = getStoryName("Stylers");
+
+export default {
+  title: `${storyName}/Sass`,
+};
+
+export const mixins = () => (
+  <Story className="story--stylers">
+    <h1>Mixin Examples</h1>
+
+    <h4>
+      <code>@truncate-text</code>
```

Comment:
These seem misleading since the `@` is on `@include`, not the mixin name (`@include truncate-text;`). Again, for people less familiar with the syntax, having an actual copy-pasteable bit would be nice. I know it's mentioned in `Docs` but just looking at the story you're not seeing that.